specs/latest/1.0/index.html

```diff
@@ -395,6 +395,38 @@ <h3><a name="THE_DRAWING_BUFFER">The Drawing Buffer</a></h3>
         </p>
     </div>
 
+    <dl class="methods">
+        <dt class="idl-code">undefined backbufferStorage(GLenum sizedFormat, unsigned long width, unsigned long height);
+        <dd>
+            Respecify the size and format of the backbuffer,
+            similar to <code>renderbufferStorage</code> as used on Renderbuffers.
+
+            Clearing behavior is equivalent to setting <code>HTMLCanvasElement.width</code> and <code>HTMLCanvasElement.height</code>, followed by changing the backbuffer format.
+            This method respects <code>WebGLContextAttributes.antialias</code>.
+
+            <div class="note">
+                <code>backbufferStorage</code> allows for efficiently respecifying width and height together simultaneously.
+                With <code>HTMLCanvasElement.width</code> and <code>HTMLCanvasElement.height</code>,
+                setting one and then the other can incur intermediate reallocations, though User Agents do try to optimize these out.
+            </div>
+
+            If <code>WebGLContextAttributes.alpha</code> is <code>false</code>, generate <code>INVALID_OPERATION</code>.
+
+            <code>RGBA8</code> is always supported for <code>sizedFormat</code>.
+            Additionally, the following enums are supported if they are currently valid for <code>renderbufferStorage</code>:
+            <ul>
+                <li><code>RGBA8</code>
+                <li><code>SRGB8_ALPHA8</code> (<code>0x8C43</code>) (E.g. from EXT_sRGB)
```

Comment:
Let's explicitly say that in a WebGL 1.0 context, the EXT_sRGB extension must be enabled in order to use this format, and that in WebGL 2.0 contexts, it's always supported.

Reply:
I'll leave a note but this is sufficiently precise as written normatively already.